build-config.json

```diff
@@ -6,10 +6,11 @@
     "srcResources": "./src/content/**/*.@(jpg|jpeg|png|gif)",
     "srcDiagrams": "./src/content/**/*.@(dot|neato|fdp|sfdp|twopi|circo)",
     "srcContentBase": "./src/content",
-    "srcAssetImages": "./src/@(assets)/**/*.@(jpg|jpeg|png|gif)",
+    "srcStaticAssets": "./src/@(assets)/**/*.@(jpg|jpeg|png|gif|ttf)",
```

Comment:
I'm now bundling an Xbox UI font released by Microsoft. It looks nice for headers.